CRAMv3.tex

```diff
@@ -13,7 +13,6 @@
 
 \usepackage{graphicx}
 \usepackage{array}
-\usepackage{ulem}
```

Comment:
The equivalent in _CRAMv2.1.tex_ has fallen off along the way. As this changes the appearance of the CRAM documents, IMHO you should make this a separate PR for the attention of the CRAM maintainers, who can decide how they want to do underlining/emphasis.

Reply:
I changed this since it didn't work well with the `latexdiff`....and it didn't seem to change the appearance that much....

Reply:
I think that we concluded that this is OK in the call.

Reply:
We accepted in the call that there is room for changing which packages are used as it was a rather arbitrary decision made by some ghastly xslt docx to tex conversion script.  It wasn't clear what the precise effect would be though, so we'd want to evaluate.

If it's just for making \emph underlined (which I must admit I was somewhat surprised to discover it did that in CRAM doc but hadn't got around to looking for out what was causing it), then I don't care about removing it.

Reply:
Update: I find my old copy of CRAMv2.1.docx and sure enough it has underline for those bits of texts.  I expect the conversion script was just trying to faithfully reproduce the same formatting, but I'm happy to be more traditional and use italics in such cases.

Reply:
I've gone through all the \emphs in the two CRAM documents and have a PR coming soon to do something appropriate for each of them and then nuke ulem.

Reply:
Thanks.  I see one of them at least is a citation.  Do we have a recommended way of doing this?  I spotted the Peter Cock one in SAMv1.tex, but that is also just simulated via combinations of textit and emph.  I've done something similar for my fledgeling CRAMv4.tex document.

Reply:
There's several in the SAM specs, and they use `{\bf }{\sl }\href{URL}{doi:…}` etc to hand-roll a standard citation format (as a footnote).

The current draft of the crypt4gh spec uses bibtex with full information on the referenced works in a _references.bib_ file. If that ever lands in hts-specs I might be tempted to convert to that method and join them.

Reply:
I wouldn't object to going down the bibtex route, but happy to go with the current flow until it becomes a major concern.

Reply:
This is now PR #373.